index.html

```diff
@@ -3,9 +3,24 @@
     <head>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+            rel="stylesheet"
+            href="C:\Users\Carte\github-sample\community-html-css-page-1\Styles sheet Community HTML CSS.css" />
         <title>Document</title>
     </head>
     <body>
-        <h1>This is some text.</h1>
+        <h1>html and css project.</h1>
+        <br />
+        <section class="photo_section"></section>
+
+        <img
+            src="https://github.com/rodgtr1/community-html-css-page/blob/main/images/45.png"
```

Comment:
```suggestion
            src="/images/45.png"
```
can also use relative paths for these